pkg/ui/src/redux/analytics.ts

```diff
@@ -26,8 +26,24 @@ const defaultRedactions = [
         match: new RegExp("/database/.*/table/.*"),
         replace: "/database/[db]/table/[tbl]",
     },
+    // The clusterviz map page, which puts localities in the URL.
+    {
+        match: new RegExp("/overview/map((/.+)+)"),
+        useFunction: true, // I hate TypeScript.
+        replace: function countTiers(original: string, localities: string) {
+            const tierCount = localities.match(new RegExp("/", "g")).length;
+            let redactedLocalities = "";
+            for (let i = 0; i < tierCount; i++) {
+                redactedLocalities += "/[locality]";
+            }
+            return original.replace(localities, redactedLocalities);
+        },
+    },
```

Comment:
nit: weird; this file is the only one I've seen that's (mostly) indented with 4 spaces. If you feel like it, add on another commit that reindents to two.

Reply:
didn't want to worry about it at the moment... i'm thinking we should stop dealing with these things manually.  thoughts about moving to prettier?

Reply:
I’m up for giving it a try. I know @BramGruneir did earlier and it did a bunch of weird things; may take some config.